bases/br_fbsp_absp/capital/table_config.yaml

```diff
@@ -0,0 +1,573 @@
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_fbsp_absp
+
+table_id: capital
+
+# Título da tabela.
+title: Estatísticas Criminais por Capitais e Distrito Federal
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: Trata-se de Estatísticas Criminais por capitais e Distrito Federal
```

Comment:
Sugiro o aprimoramento aqui: A base contempla as informações do Anuário de Segurança Pública a partir de 2016 com as estatísticas criminais compatibilizadas entre anos, com informações acerca dos homicídios, tráfico, furto e rouba, além da força policial, entre outros ao nível das capitais brasileiras e distrito federal.